src/github/client/github-client-errors.ts

```diff
@@ -68,10 +70,13 @@ export class GithubClientGraphQLError extends GithubClientError {
 		super(message);
 		this.errors = errors;
 	}
-
 }
 
-export const isChangedFilesError = (err: GithubClientGraphQLError): boolean =>
-	!!err?.errors?.find(e => e.message?.includes("changedFiles"));
+const logger = getLogger("github.errors");
+export const isChangedFilesError = (err: GithubClientGraphQLError | GithubClientError): boolean => {
+	logger.warn({error: err}, "isChangedFilesError");
```

Comment:
Update the warning message to the new checks

Reply:
not a bad idea.